File: Server/Models/Sediste.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;

namespace Server.Models
{
    [Table("Sediste")]
    public class Sediste
    {
        [Column("Broj")]
        [Key]
        public int broj {get; set;}

        [Column("Zauzetost")]
        public bool zauzetost {get; set;}

        [Column("uSali")]
        public virtual List<SalaSediste> Veza { get; set; }

        [Column("FormaBioskopaID")]
        public int FormaBioskopaID { get; set; }
    }
}

File: Client/JS/sediste.js
export class Sediste {

    constructor(broj, zauzetost) {
        this.broj = broj;
        this.zauzetost = zauzetost;
        //true - zauzeto
        //false - slobodno
    }

    get Broj(){
        return this.broj;
    }

    set Broj(value){
        this.broj = value;
    }

    get Zauzetost() {
        return this.zauzetost;
    }

    set Zauzetost(value) {
        this.zauzetost = value;
    }
}